index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<main id="main">
+  <head> 
+    <meta charset="UTF-8">
+    <title >FCC: Tribute page</title>
+   
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <h1 id = "title"> Dr. Norman Borlaug </h1>
+    <p id="title">
```

Comment:
- [ ] More than one element should not have the same id, in this case,` id="title"`. `id` is a unique identifier for elements. If you wish to give a group of elements the same property, you should try using a class instead.
- [ ] Only the h1 element is supposed to have an `id="title"` as per the requirements for this project.